common/lc_http.cpp

```diff
@@ -81,7 +81,10 @@ lcHttpManager::lcHttpManager(QObject* Owner)
 
 lcHttpReply* lcHttpManager::lcHttpManager::DownloadFile(const QString& Url)
 {
-	return (lcHttpReply*)get(QNetworkRequest(QUrl(Url)));
+    QNetworkRequest req = QNetworkRequest(QUrl(Url));
+    req.setAttribute(QNetworkRequest::RedirectPolicyAttribute, QNetworkRequest::NoLessSafeRedirectPolicy);
```

Comment:
`req.setAttribute(QNetworkRequest::RedirectPolicyAttribute, QNetworkRequest::NoLessSafeRedirectPolicy);` this was introduced in Qt 5.9.0 and LeoCAD's minimum Qt version is 5.4 so you should consider adding
~~~cpp
#if (QT_VERSION >= QT_VERSION_CHECK(5, 9, 0))
    req.setAttribute(QNetworkRequest::RedirectPolicyAttribute, QNetworkRequest::NoLessSafeRedirectPolicy);
#endif
~~~

Cheers,